packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-subgraph/index.esm.t

```diff
@@ -0,0 +1,43 @@
+---
+to: "<%= rootDir %>/<%= h.changeCase.param(name) %>/index.ts"
+force: true
+---
+import { Subgraph } from "@powerhousedao/reactor-api";
+import { Knex } from "knex";
+import { gql } from "graphql-tag";
+
+export class <%= pascalName %>Subgraph extends Subgraph {
+  name = "<%= h.changeCase.param(name) %>";
+  resolvers = {
+    Query: {
+      example: {
+        resolve: async (parent, args, context, info) => {
+          return "example";
+        }
+      }
+    }
+  }
+
+  typeDefs = gql`
+    type Query {
+      example(id: ID!): String
+    }
+  `;
+
+  additionalContextFields = {
```

Comment:
Feature request:
Add support to adding context per request. The subgraph would have access to an incoming request and could set the context dynamically.

Reply:
nice idea, but lets do this in the next pr